scripts/dev/docker-compose.yml

```diff
@@ -56,7 +56,21 @@ services:
     ports:
       - "3000:80"
     depends_on:
-      - nginx
+      - user
```

Comment:
The admin container dependencies should not be changed and should depend on Nginx for its startup.